sigstore/_internal/fulcio/client.py

```diff
@@ -83,71 +84,69 @@ class FulcioSCTError(Exception):
     pass
 
 
-class FulcioSignedCertificateTimestamp(SignedCertificateTimestamp):
-    def __init__(self, b64_encoded_sct: str):
-        self.struct = json.loads(base64.b64decode(b64_encoded_sct).decode())
+class FulcioSCT(BaseModel):
```

Comment:
N.B.: We could probably rename this to something like `DetachedFulcioSCT` once we merge embedded SCT support.

Reply:
Nope, no reason. I can rename.